mods/ra/rules/fakes.yaml

```diff
@@ -39,8 +39,8 @@ WEAF:
 	Bib:
 	RenderBuilding:
 		Image: WEAP
-	WithIdleOverlay@ROOF:
-		Sequence: idle-top
+	WithProductionDoorOverlay:
```

Comment:
Or just revert partly https://github.com/Mailaender/OpenRA/commit/8f93d7b5d76a20174349830016a6ada05cb27ee4 to bring back `idle-top`.